endpoints.json

```diff
@@ -23,5 +23,74 @@
         }
       ]
     }
+  },
+  "GET /api/users":{
+    "description":"serves an array of users",
+    "queries":[],
+    "exampleResponse":{
+      "users":[{
+      "username": "butter_bridge",
+      "name": "jonny",
+      "avatar_url":
+        "https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg"
+    }]}
+  },
+  "GET /api/articles/:article_id":{
+    "description":"serves an article object",
+    "queries": ["article_id"],
+    "exampleResponse":
+      {
+        "article_id":1,
+        "title": "Living in the shadow of a great man",
+        "topic": "mitch",
+        "author": "butter_bridge",
+        "body": "I find this existence challenging",
+        "created_at": 1594329060000,
+        "votes": 100,
+        "comment_count":11
+      }
+  },
+  "GET /api/articles/:article_id/comments":{
+    "description":"serves an array of comments for an article",
+    "queries": ["article_id"],
+    "exampleResponse":{
+      "comments":[
+      {
+        "comment_id":5,
+        "article_id":1,
+        "author": "butter_bridge",
+        "body": "text",
+        "created_at": 1594329060000,
+        "votes": 100
+      }]}
+  },
+  "POST /api/articles/:article_id/comment":{
+    "description":"posts a comment relating to article",
+    "queries":["article_id"],
+    "examplePost":{
```

Comment:
This can be changed to exampleBody, as all of these properties would be within the body sent by the client.